cypher/cypher-docs/src/docs/dev/deprecations-additions-and-compatibility.asciidoc

```diff
@@ -729,6 +729,22 @@ a|
 Not a syntax change but a semantic one.
 The `EXECUTE BOOSTED` privilege will no longer include an implicit `EXECUTE` privilege when granted.
 That means that to execute a procedure or a function with boosted privileges both `EXECUTE` and `EXECUTE BOOSTED` are needed.
+
+a|
+label:functionality[]
+label:updated[]
+[source, cypher, role="noheader"]
+----
+SHOW DATABASES YIELD *
+----
+a|
+New columns have been added the result:
```

Comment:
This sentence is a bit confusing. Did you mean adding new columns to the result or as the result, for instance?

Reply:
Yes, it is. I've added 'to'